code/datums/antagonists/datum_devil.dm

```diff
@@ -387,8 +363,8 @@ var/global/list/lawlorify = list (
 						return 1
 				return 0
 
-/datum/devilinfo/proc/hellish_resurrection(mob/living/body)
-	message_admins("[owner.name] (true name is: [truename]) is resurrecting using hellish energy.</a>")
+/datum/antagonist/devil/proc/hellish_resurrection(mob/living/body)
+	message_admins("[owner.mind.name] (true name is: [truename]) is resurrecting using hellish energy.</a>")
```

Comment:
Actually, that won't work either. You may need to significantly rework this, given that antag datums don't stick around with no mob.